NS_0100A6301214E000_Fire_Emblem_Engage.js

```diff
@@ -0,0 +1,44 @@
+// ==UserScript==
+// @name         [0100A6301214E000] Fire Emblem Engage
+// @version      1.3.0 - 1.3.0
+// @author       [Sphyralg zx96]
+// @description  Yuzu
+// * Intelligent Systems Co., Ltd
+// * Unity (il2cpp)
+//
+// KnowIssue: No support for 1.0.0 (yet)
+// ==/UserScript==
+const gameVer = '1.3.0';
+
+const { setHook } = require('./libYuzu.js');
+
+const mainHandler = trans.send(handler, -200); // db leading prevemt double
```

Comment:
const mainHandler = trans.send(handler, "200+"); Better if there's other hook to add later(subtitles etc)..

Reply:
Good point.
I would like to try and add extra hooks for subtitles and some additional non-dialogue menus when I get the chance.